frontend/packages/local-storage-operator-plugin/src/components/disks-list/disks-list-page.tsx

```diff
@@ -96,6 +100,7 @@ const DisksList: React.FC<TableProps> = (props) => {
   return (
     <Table
       {...props}
+      loadError={props.customData.errorMessage}
```

Comment:
We dont want to miss out firehose component's  loading errors - > `props.loadError`.
Please add a `label` prop as well , to provide a descriptive error message -> `Error loading {{label}}`
```suggestion
      loadError={props.customData.error || props.loadError}
      loaded={props.custom.loaded && props.loaded}
      label="Disks"
```